### PR TITLE
Guardian/Parasite HUD and rebalancing [Ready for review]

### LINKED ISSUE #12344: Guardian users retain their guardian verbs even if their Guardian dies
If they get revived after Guardian death they will still have a communication verb.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Dying with a parasite now turns you to dust.

Fixes #12344

Damage transferred to you by your parasite is doubled while you are in crit, with half of it being cloneloss.

The Chaos parasite now only ignites you if you are not already on fire, meaning it wont rapidly stack fire on you until its impossible to put out. It now has a slightly higher chance to teleport you on hit though.

The ranged parasites projectiles now bypass armour.

The Explosive types traps now are on a 20 second cooldown

The messages for the user taking damage from the transfer are now bolded, so they should be a bit easier to see in combat.

Adds the ability to toggle lights on/off (glowing).

Adds HUD buttons for manifest, recall, communicate, toggle, and toggle lights.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
so you're going with the high risk/high reward method of balance

i like it

**Comment by KorPhaeron** [comment]
@PKPenguin321 

Not even sure if they're high reward currently. Earlier I watched a clown kill a stand user by running away and harm batoning him while the stand failed to catch up. Granted the stand probably should have manifested before the cuffs went on, but still.

**Comment by Durkel** [comment]
To be totally honest, if you die with as a traitor with a parasite you're not going to get cloned anyway.

 I don't even see why these garner so much negative attention, whenever you buy a parasite you roll the dice on a partner you know nothing about. In fact you have a higher chance of getting someone who is totally going to screw you over than a good player. People need to buck the fuck up and stop complaining about dying.

**Comment by PKPenguin321** [comment]
>Earlier I watched a clown kill a stand user by running away and harm batoning him while the stand failed to catch up

this clown was me, actually

**Comment by Lumipharon** [comment]
loot explodes with the host - RIP.

**Comment by KorPhaeron** [comment]
I guess I could make them drop all items first

**Comment by PKPenguin321** [comment]
that would be cool
give it a message like "Traitor McGee's body vanishes in a flash, leaving only his belongings and empty clothes behind..."

**Comment by KorPhaeron** [comment]
@Durkel 

I've actually seen enough people get cloned after death with a parasite (because the parasite often kills the attacker while its owner is in crit, and the whole pile of bodies is taken to medbay in the confusion) that I was considering giving them a button to regenerate their parasite if they get revived.

I dunno why there is such a negative reaction to them balance wise (especially when changelings exist), but if I don't nerf them someone else is going to (or remove them all together).

**Comment by Bawhoppen** [comment]
why do you want to buff stands even more? now you get a free explosive implant built in?

**Comment by KorPhaeron** [comment]
>being taken out of the round permanently is a buff

**Comment by Bawhoppen** [comment]
why is explosive implant even an item with your logic

**Comment by KorPhaeron** [comment]
Because it turns you into a bomb and denies all your items (which only really matters in nuke) I already said 3 hours ago I'd make it drop items for the stand.

**Comment by Nanotransistor** [comment]
Only problem i have with parasites is that they are so cheap. They should cost either all of your TC or at least 17. Some people say there is a lot of luck and rng involved, but so is with the surplus crate, and that one cost 20.

**Comment by Bawhoppen** [comment]
@Nanotransistor Thats what I've been saying, but I keep being shot down. I compromised on 16 TC so you can still buy SOMETHING.

**Comment by KorPhaeron** [comment]
@Nanotransistor

The surplus crate gives you 50tc worth of shit to reward you for gambling so thats not really a great example.

The parasites in their current state simply are not strong enough nor flexible enough to be worth spending all your crystals on either.

**Comment by Bawhoppen** [comment]
what do you mean they're not strong enough? they definitely are

**Comment by Nanotransistor** [comment]
@KorPhaeron Which is why i say to increase it a bit. The other day i had a guy cuffed, had him in the brig and then out of nowhere he pulls out a parasite, that kills me just by bumping into me, since that fire was impossible to put out. I never thought he may have a parasite since he had been emagging borgs left and right. Then i learned the parasite costs a mere 12 TC.

Since it is a really strong item that you can use even while cuffed, at least the price should go up by a decent amount. You don't see traitors buying both a revolver and an esword. 

Also it seems like it is the only item most traitors are buying. Wouldn't like for traitor to become just Parasite the gamemode.

**Comment by Cookiesndip** [comment]
Holoparasites are useless, even for the cost. You roll the die for who you get, and the average skill level on the server is very very low. What do you get for that 12 TC? A second, slow-moving target that transfers all damage to you, so you're twice as vulnerable, a person of unknown quality who may or may not do what you want them to do in the heat of combat or get you killed, offering minor utility/gimmick (deathtrap teleport etc.), most of which can be essentially accomplished with onstation materials easily (flamethrower, beacon teleporter to deathtrap + handtele or disposals rerouting, etc.), and the inability to buy a real weapon yourself, who you can actually rely on to do what you want them to do.

Hell, the hard-counter for stands? Ignore the stand, stun the host and drag them away while beating them to death. Even if the stand recalls and manifests, you're going too fast for them to hit you so long as you don't hit any doors. Keep in mind there's a slight delay in manifestation where the camera is ajar as well, so it's even harder to hit you. This has never failed me yet, even when I'm partially wounded, and very rarely do stand users even survive into the mid-lategame unless they didn't buy equipment until later.

I bought holoparasites a few times and never bought them again. They're flat-out not worth 12 TC, and have a crazy number of hard counters. I'd buy adrenals, an emag, or hell, even thermals over it, even if they were all 12 TC. An ebow has a gajillion times the utility as a stand, and costs the same. Hell - 'running away' is a totally viable strategy against one, esp. since the user can't buy a ranged weapon.

**Comment by Bawhoppen** [comment]
Observing every traitor round would seem to show differently.

**Comment by KorPhaeron** [comment]
@Bawhoppen 

When I observe the vast majority of them die to either  gunfire or in the way @Cookiesndip describes, I dunno what rounds you're observing.

@Nanotransistor 

It's why I'm nerfing the fire one so you can actually put yourself out. I'd rather make them weaker and have them be partners to traitors than replace all of a traitors gear. Not much of a teamwork dynamic if one guy has nothing and is just the babysitter.

**Comment by KorPhaeron** [comment]
@Dorsisdwarf @Razharas  

Ready for review

**Comment by Ricotez** [comment]
maybe turn the user into a pile of ashes instead of gibbing them? or into a pile of bones like soul gems do. would also explain why their stuff stays behind

**Comment by Anonus** [comment]
The verbs are really finicky sometimes, does making them buttons fix that?
Finicky as in you click to recall/manifest and nothing happens, even if there's no lag going on.

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -33,18 +34,23 @@
 	var/magic_fluff_string = " You draw the Coder, symbolizing bugs and errors. This shouldn't happen! Submit a bug report!"
 	var/tech_fluff_string = "BOOT SEQUENCE COMPLETE. ERROR MODULE LOADED. THIS SHOULDN'T HAPPEN. Submit a bug report!"
 	var/bio_fluff_string = "Your scarabs fail to mutate. This shouldn't happen! Submit a bug report!"
-
+/*
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Why is this commented.

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Easier to test they dont die immediately on spawning. Knew I'd forget to uncomment eventually

Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Ah that does make sense.
make sure to commit the undo.

**Comment by KorPhaeron** [comment]
@anonus

Hopefully. Its hard to reproduce it locally.

Theres also a 3 second delay every time you manifest or recall so that might be it

**Comment by KorPhaeron** [comment]
@Ricotez

I was gonna use a skeleton but then I realized it was technically possible for nonhuman things to have a stand.

Maybe I'll just do a skeleton anyway.

**Comment by KorPhaeron** [comment]
@RemieRichards 

Comment is fixed, ready to be merged

**Comment by Ricotez** [comment]
@KorPhaeron if you don't want to use gibs but the pile of bones is a problem just use ashes or a pile of dust or something

**Comment by KorPhaeron** [comment]
Oh there's a dust proc anyway that handles everything

**Comment by KorPhaeron** [comment]
@Miauw62 

I nerfed parasites you should merge it

**Pull request closed without merging**